pandas/core/arrays/arrow/array.py

```diff
@@ -2466,9 +2466,9 @@ def _round_temporally(
             "W": "week",
             "D": "day",
             "H": "hour",
-            "T": "minute",
+            "min": "minute",
             "S": "second",
-            "L": "millisecond",
+            "ms": "millisecond",
             "U": "microsecond",
```

Comment:
should we change "U" to "us" and "N" to "ns" etc while we're doing it?  seems weird to make "ms" an outlier

Reply:
Thank you for the comment. I agree, they look like outliers to me as well. I think we can change `"U" `to `"us"` and `"N"` to `"ns"`, I'm not sure about other aliases thought. Can I do it in a separate PR?